rpc/query_server.go

```diff
@@ -1136,7 +1157,7 @@ func getReceiptByTendermintHash(state loomchain.State, blockStore store.BlockSto
 	}
 	txHash, err := eth.DecDataToBytes(txObj.Hash)
 	if err != nil {
-		return eth.TxObjToReceipt(txObj, contractAddr), errors.Wrapf(err, "invalid loom transaction hash %h", txObj.Hash)
+		return eth.TxObjToReceipt(txObj, contractAddr), errors.Wrapf(err, "invalid loom transaction hash %v", txObj.Hash)
```

Comment:
Is this a merge error?

Reply:
Whenever I run `rpc/query_server_test.go` I always got the error `Wrapf format %h has unknown verb h` and test build failed 
So should I remain using %h or change it to %X instead?

Reply:
It should be `%x` instead of `%h`